ocp_resources/node_network_configuration_enactment.py

```diff
@@ -3,3 +3,18 @@
 
 class NodeNetworkConfigurationEnactment(Resource):
     api_group = Resource.ApiGroup.NMSTATE_IO
+
+    class Conditions:
+        class Type:
+            FAILING = "Failing"
+            AVAILABLE = "Available"
+            PROGRESSING = "Progressing"
+            PENDING = "Pending"
+            ABORTED = "Aborted"
+
+        class Reason:
+            CONFIGURING = "ConfigurationProgressing"
```

Comment:
Please match the constant name to the actual reason.
CONFIGURATION_PROGERESSING = "ConfigurationProgressing"
and so on

Reply:
this should be for both files(Enactment + policy) or just Enactment?

Reply:
for both, thx.

Reply:
Done